### ISSUE TITLE
How to set dark mode?

### ISSUE THREAD

**Issue by ToasterUwU** [flathub/io.github.pyfa_org.Pyfa#12]
Thanks for making this, i always prefer Flatpaks over Appimages.

One issue i have and cant seem to figure out is that the `~/.config/gtk-3.0/settings.ini` doesnt seem to affect the flatpak version.
The Appimage version will see the `gtk-application-prefer-dark-theme=true` setting in that file and then load in dark mode, but the flatpak version either ignores it or doesnt have access to it.

I have very light-sensitive eyes, so i cant use the stock light theme version without extreme eyestrain after a short while.

If you have any ideas how to fix this, that would be greatly appericated

**Comment by asbytes** [comment]
You can copy `settings.ini` to `~/.var/app/io.github.pyfa_org.Pyfa/config/gtk-3.0`

**Comment by ToasterUwU** [comment]
I already have the same setting in that place as well, but it doesnt do anything. The app still starts in light mode :/

**Comment by asbytes** [comment]
This works for me:
> You can copy `settings.ini` to `~/.var/app/io.github.pyfa_org.Pyfa/config/gtk-3.0`

You could try to inspect app overrides with Flatseal. Maybe there are some excessive environment variables or file system permissions.

**Comment by ToasterUwU** [comment]
Weird.. I don't see any overrides or environment variables that are set and could do anything like that. 

Unless you have another idea, im pretty much out of them.. If this is just a me issue i can just stick with the Appimage and close this issue.

**Comment by arrowmaster** [comment]
I previously had dark mode working on Debian KDE Plasma5 using the following but it has stopped working after upgrading to Plasma6. A different system on Debian KDE Plasma5 still has working dark mode.

`flatpak override --user --filesystem=xdg-config/gtk-3.0:ro`

**Comment by ToasterUwU** [comment]
mhh yeah, i am on Plasma 6, so it wouldnt help me sadly. But thats fine, i since switched to the newly added Nix package

**Comment by arrowmaster** [comment]
I finally found an upstream bugreport for Plasma6 and a solution https://bugs.kde.org/show_bug.cgi?id=485028

TLDR `flatpak override --user --env=GTK_THEME=Breeze io.github.pyfa_org.Pyfa`

**Comment by ToasterUwU** [comment]
Wonderful, that should do it for everyone who wants dark mode
